src/main/java/WayofTime/bloodmagic/item/ItemLavaCrystal.java

```diff
@@ -73,6 +83,27 @@ public Binding getBinding(ItemStack stack) {
         return new Binding(NBTUtil.getUUIDFromTag(nbt.getCompoundTag("id")), nbt.getString("name"));
     }
 
+    @Override
+    public EnumActionResult onItemUse(EntityPlayer player, World worldIn, BlockPos pos, EnumHand hand, EnumFacing facing, float hitX, float hitY, float hitZ) {
+        pos = pos.offset(facing);
+        ItemStack itemstack = player.getHeldItem(hand);
+
+        if (!player.canPlayerEdit(pos, facing, itemstack))
+            return EnumActionResult.FAIL;
+
+        else {
```

Comment:
Nesting this in an `else` is unnecessary since it'll never be reached if the `if` passes.